lms/envs/bok_choy.py

```diff
@@ -124,9 +124,6 @@
 # Enable custom content licensing
 FEATURES['LICENSING'] = True
 
-# Enable persistent subsection grades, so that feature can be tested.
-FEATURES['ENABLE_SUBSECTION_GRADES_SAVED'] = True
```

Comment:
So how is this feature now enabled for all bokchoy and unit tests, by default?
We still want to run all tests with the new grades code.
This must explain why the tests didn't catch the django "admin" folder issue.